.changeset/warm-seas-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": major
```

Comment:
Hi @atmd83,

Could you change "@ledgerhq/live-common" from **major** to **patch** ?

Reply:
@aelmilligy-ledger done 👍